### ISSUE TITLE
Can't import the package

### ISSUE THREAD

**Issue by capem** [np-8/dash-uploader#9]
```
File "C:\Users\****\Miniconda3\envs\dash-env\lib\site-packages\dash_uploader\configure_upload.py", line 8, in <module>
    from dash.dependencies import Input, Output, State, ALL
ImportError: cannot import name 'ALL' from 'dash.dependencies' (C:\Users\****\Miniconda3\envs\dash-env\lib\site-packages\dash\dependencies.py)
```

**Comment by np-8** [comment]
Thanks for reporting. What is the output of 

```python
import dash
print(dash.__version__)
```

i.e. what is the version of `dash` you are using?

**Comment by capem** [comment]
import dash
print(dash.__version__)
1.9.1

**Comment by capem** [comment]
Well I just removed the ALL from the imports in the 'configure_upload.py' file and it worked.

**Comment by np-8** [comment]
Yeah seems that [`ALL` was added to `dash.dependencies` at version 1.11.0](https://github.com/plotly/dash/blob/dev/CHANGELOG.md#1110---2020-04-10). The import was unnecessary so I just removed it.

This is now fixed in dash-uploader 0.3.1.
